wiki/Client/Beatmap_editor/Beat_snap_divisor/de.md

```diff
@@ -1,59 +1,34 @@
 ---
-outdated_translation: true
+tags:
+  - bsd
 ---
 
-# Beat Snap Divisor
+# Taktteiler
 
-## osu!, osu!taiko und osu!catch
+![](img/beat_snap_divisor-DE.png "Taktteiler im Editor")
 
-Der Beat Snap Divisor wird benutzt, um die Positionen, auf denen Noten gelegt werden können, zu limitieren. Um die Zählzeit zu erhöhen, bewegen Sie einfach den Balken nach rechts.
+Der **Taktteiler** (auch *Beat Snap Divisor* genannt) ist eine Einstellung, die bestimmt, wie der Editor [Beats](/wiki/Music_theory/Beat) darstellt. Dieser Prozess nennt sich [Beat-Positionierung](/wiki/Beatmapping/Beat_snapping). Der Taktteiler befindet sich in der rechten oberen Ecke des Editors.
```

Comment:
Seems sensible to mention "beat snapping" since we also mention "Beat Snap Divisor".

```suggestion
Der **Taktteiler** (auch *Beat Snap Divisor* genannt) ist eine Einstellung, die bestimmt, wie der Editor [Beats](/wiki/Music_theory/Beat) darstellt. Dieser Prozess nennt sich [Beat-Positionierung](/wiki/Beatmapping/Beat_snapping) (engl. **beat snapping**). Der Taktteiler befindet sich in der rechten oberen Ecke des Editors.
```